src/containers/settings/index.tsx

```diff
@@ -1,79 +1,71 @@
+  width: 100%;
+  min-height: 3rem;
+  margin-bottom: 1rem;
+`;
 
+const Settings: React.FC = () => {
   return (
     <>
       <Helmet>
         <title>Settings</title>
-        <meta name="description" content="Description goes here." />
+        <meta
+          name="description"
+          content="Settings for Lucy's Love Bus Programs account."
+        />
       </Helmet>
       <ContentContainer>
-        <Title>Settings</Title>
-
-        <Form name="basic" onFinish={onFinishChangePassword}>
-          <Form.Item
-            label="Current Password"
-            name="currentPassword"
-            rules={[
-              {
-                required: true,
-                message: 'Please input your current password!',
-              },
-            ]}
-          >
-            <Input.Password />
-          </Form.Item>
-          <Form.Item
-            label="New Password"
-            name="newPassword"
-            rules={[
-              { required: true, message: 'Please input your new password!' },
-            ]}
-          >
-            <Input.Password />
-          </Form.Item>
-          <Form.Item
-            name="confirm"
-            label="Confirm Password"
-            dependencies={['newPassword']}
-            hasFeedback
-            rules={[
-              {
-                required: true,
-                message: 'Please confirm your password!',
-              },
-              ({ getFieldValue }) => ({
-                validator(_, value) {
-                  if (!value || getFieldValue('password') === value) {
-                    return Promise.resolve();
-                  }
-                  return Promise.reject(
-                    'The two passwords that you entered do not match!',
-                  );
-                },
-              }),
-            ]}
-          >
-            <Input.Password />
-          </Form.Item>
-          <Form.Item>
-            <Button type="primary" htmlType="submit">
-              Change Password
-            </Button>
-          </Form.Item>
-        </Form>
+        <CenteredOrangeTitle>Settings</CenteredOrangeTitle>
+        <CenteredButtonContainer>
+          <SettingsButton to={Routes.CHANGE_ACCOUNT_EMAIL}>
+            Change Primary Account Email
+            <MailOutlined />
+          </SettingsButton>
+          <SettingsButton to={Routes.CHANGE_PASSWORD}>
+            Change Password
+            <KeyOutlined />
+          </SettingsButton>
+          <SettingsButton to={Routes.SET_CONTACTS}>
+            Edit Account Information
+            <EditOutlined />
+          </SettingsButton>
+          <SettingsButton to={Routes.PERSONAL_REQUESTS}>
+            Request to Become a Participating Family
+            <UsergroupAddOutlined />
+          </SettingsButton>
+          <SettingsButton to={Routes.DEACTIVATE_ACCOUNT}>
+            Deactivate Account
+            <ExclamationCircleOutlined />
+          </SettingsButton>
```

Comment:
Consider defining an object mapping the text to `Routes`, and then just map over it to create the `SettingsButton`. 
It'll be cleaner and easy to add/remove buttons.

Reply:
Its actually less lines of code to not define the data elsewhere, so I'll leave it as it is